src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -191,7 +191,7 @@ public View NavigationContent
         /// SetTitle sets title text to the default title content.
         /// If TitleContent is not TextLabel, then Title does not set title text of the TitleContent.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
SetTitle?

Reply:
Thank you for your comment.
It has been fixed.